source/demo-code/recipe-info-rough/recipePage.html

```diff
@@ -29,41 +29,106 @@
       </nav>
     <main>
         <section id="header">
+            <button style="background-color: #F1F7F7">Return</button>
```

Comment:
try to avoid inline css as much as possible for the sake of separating code. i would change to a class name like "back-btn" and move the code into the css file

Reply:
Done